packages/es-components/src/components/containers/fieldset/Fieldset.js

```diff
@@ -43,13 +44,15 @@ Fieldset.propTypes = {
   legendContent: PropTypes.node,
   /** Additional classes to be applied to the legend element */
   legendClasses: PropTypes.string,
-  children: PropTypes.node
+  children: PropTypes.node,
+  className: PropTypes.string
```

Comment:
It would be helpful to add a description here explaining which element the className will be applied to.

Reply:
Fair, it seemed implicit when working with the fieldset that className would get passed to the Fieldset but I can add that description.

Reply:
Oh, yeah you're right, this is fine.